presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -190,6 +191,30 @@ public void dropTable(SchemaTableName tableName)
         tableCache.invalidate(tableName);
     }
 
+    public void addColumn(SchemaTableName schemaTableName, ColumnMetadata columnMetadata)
+    {
+        Document metadata = getTableMetadata(schemaTableName);
+
+        List<Document> columns = new ArrayList<>(getColumnMetadata(metadata));
+
+        Document newColumn = new Document();
+        newColumn.append(FIELDS_NAME_KEY, columnMetadata.getName());
+        newColumn.append(FIELDS_TYPE_KEY, columnMetadata.getType().toString());
```

Comment:
```suggestion
        newColumn.append(FIELDS_TYPE_KEY, columnMetadata.getTypeSignature().toString());
```
Like we handle in `MongoColumnHandle`. 
Can we have a validation for the types that are compatible with MongoDB ?   (Maybe in a different PR)

Reply:
Fixed and filed new issue for validation https://github.com/prestosql/presto/issues/5513.